lib/template-mixins.js

```diff
@@ -187,8 +187,20 @@ module.exports = function (options, deprecated) {
                     return template.render(Object.assign({
                         renderMixin: function () {
                             return function () {
-                                if (this.child && this[this.child]) {
-                                    return this[this.child]().call(this, this.toggle);
+                                if (this.child) {
+                                    if (this.child === 'html') {
+                                        try {
+                                            var key = this.toggle;
+                                            return res.locals.fields.find(function (field) {
+                                                return field.key === key;
+                                            }).html;
+                                        } catch (err) {
+                                            next(new Error('Error: html property not set on field: ' + this.toggle + '. Did you forget to use a component?'));
+                                        }
+                                    }
+                                    if (this[this.child]) {
+                                        return this[this.child]().call(this, this.toggle);
+                                    }
```

Comment:
I feel like this is perhaps getting a little deep. Might be worth pulling the child rendering out into its own function.